docs/source/reference/pruners.rst

```diff
@@ -1,3 +1,5 @@
+.. _pruners:
```

Comment:
Sorry, by [this PR](https://github.com/optuna/optuna/pull/1873), you can use module directives now. 🙏

Reply:
Thanks